Sources/Amplitude/AMPStorage.m

```diff
@@ -0,0 +1,125 @@
+//
+//  Amplitude.m
+//  Copyright (c) 2013 Amplitude Inc. (https://amplitude.com/)
```

Comment:
same question for the year here. And do we plan to add some test to cover this file?